tests/suites/target_test.function

```diff
@@ -71,10 +90,10 @@ int verify_dependencies( uint8_t count, uint8_t * dep_p )
 uint32_t receive_uint32()
 {
     uint32_t value;
-    value =  (uint8_t)greentea_getc() << 24;
-    value |= (uint8_t)greentea_getc() << 16;
-    value |= (uint8_t)greentea_getc() << 8;
-    value |= (uint8_t)greentea_getc();
+    value =  receive_byte() << 24;
```

Comment:
Instead of calling `receive_byte` 4 times, how about calling `receive_hexdigit` 8 times?

Reply:
Why? What's the added value?  Calling `receive_hexdigit` 8 times is equivalent to calling `greentea_getc()` 8 times, and then call `assert( unhexify( &byte, &c ) != 8 );`